.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--
+Label the PR with the kind of change this for:
+
+kind/feature
+kind/bug
+kind/documentation
+kind/regression
+kind/*
+-->
```

Comment:
we don't have a prow running in the repo so these labels won't be recognized by GitHub, but we can leave them maybe as "reminder" to label the PR